documentation/src/main/asciidoc/querylanguage/From.adoc

```diff
@@ -131,7 +131,7 @@ select id, total
 from (
     select ord.id as id, sum(item.book.price * item.quantity) as total
     from Order as ord
-        join Item as item
+        join items as item
```

Comment:
Please revert. The original seems correct.

Reply:
I tested it locally and it ended up with runtime exception. Joining an Entity needs further condition (on... or where ...), but joining attribute is not ambiguious and works as expected.

Reply:
Ughh, I'm sorry. I didn't realize that there was an association named `items`. I had assumed this was referring to a table name. So then, to avoid confusion, I would say it's better to write `ord.items`

Reply:
I agree. It should be `join ord.items as item`.

Reply:
I changed it to `join ord.items as item` (be consistent with other uses in the doc) and fixed another similar one I overlooked just below.

Reply:
> Ughh, I'm sorry. I didn't realize that there was an association named `items`. I had assumed this was referring to a table name. So then, to avoid confusion, I would say it's better to write `ord.items`

it doesn't matter at all. From my understanding, the whole point of code review comment is the potential 'true positive'. 'false positive' is expected and even they help to clarify for they reflect common reaction and user perspective. Code review comments are always good to have and never enough! Thanks for your comments, as always.